DESY/VBF_M125_SM_tt.txt

```diff
@@ -0,0 +1,27 @@
+/////FOR THE PURPOSE OF CP MEASUREMENT/////
+
+PATH:
+
+/afs/cern.ch/user/d/dvats/public/for_sync/VBFHToTauTau_M125_13TeV_powheg_pythia8_v1.root
+
+
+DESCRIPTION:
+
+Events are stored where both tau leptons decay hadronically. The following tauID selections are applied:
+
+tau_byVVVLooseDeepTau2017v2p1VSjet
+tau_byVVVLooseDeepTau2017v2p1VSe
+tau_byVLooseDeepTau2017v2p1VSmu
+
+The following b-tag selections are used:
+
+pfDeepCSVJetTags:probb
+pfDeepCSVJetTags:probbb
+btagCut = 0.4941
+
+The following JEC were used:
+
+jetCorrections = ('AK4PFchs', cms.vstring(['L1FastJet', 'L2Relative', 'L3Absolute', 'L2L3Residual']), 'None')
```

Comment:
I assume that JEC is re-aplied from a global tag, so it will be good to specify a global tag used or mention that original JEC is used. Regardless, it is not an issue, but it is better to have full info.

Reply:
I added the global tag.

Reply:
OK, thanks